app/src/main/java/com/soen490chrysalis/papilio/viewModel/CreateActivityViewModel.kt

```diff
@@ -56,6 +57,7 @@ class CreateActivityViewModel(private val activityRepository : IActivityReposito
         return "Number of participants must be greater than 0!"
     }
 
+    @AddTrace(name = "picture_url_validation")
```

Comment:
This trace here is unnecessary. The function is two lines long and does basic validation. Why would we need performance metrics on this function?